tools/measurement/apksize/apksize.gradle

```diff
@@ -44,6 +47,9 @@ task generateMeasurements(type: GenerateMeasurementsTask) {
  * @param -Ppull_request the pull request number to be included in the report
  */
 task uploadMeasurements(type: UploadMeasurementsTask) {
```

Comment:
FYI: We should probably have only one uploadMeasurements task in tools/measurement. We can do this in a future pull request. Then, Prow will only need to run the one task and all of our other metrics will be gathered.

Reply:
Agreed.

Also, they need to be separate at the moment, if we want to use different database config for apk size and coverage.